### ISSUE TITLE
Add dropdown menu to select different locations across US with different % eclipse coverage

### ISSUE THREAD

**Issue by patudom** [cosmicds/minids#161]
- [ ] Choose cities that will be in the list (Pat)
- [ ] Add those cities to a dropdown selector that will change the WWT lat/long.
- [ ] Default displayed city should be Albuquerque, NM
- [ ] location-selector should show the city selected from the dropdown (but should allow user to choose any city they want in the US).

**Comment by johnarban** [comment]
so @patudom  I am not sure if we can have the Location selector map show the location the user selects in the dropdown. right now it only takes the initial position as a prop. I believe @Carifio24 would need to expose the location via something like a `model-value` in order for us to be able to set it manually.

**Comment by Carifio24** [comment]
Yeah, right now the location selector is an "outgoing-only" component - it just emits the location when you choose it. I'll reconfigure it to use a two-way binding.

**Comment by Carifio24** [comment]
The updates in #168 should make that possible - that update makes it possible to do something like `v-model="locationDeg"` to get a two-way binding.

Note that if we want to use `locationDeg` from the current setup as the v-model value, it needs to be settable. This is easy to do - just replace the current definition with

```
locationDeg: {
  get(): LocationDeg {
    return {
      latitudeDeg: R2D * this.location.latitudeRad,
      longitudeDeg: R2D * this.location.longitudeRad
    };
  },
  set(value: LocationDeg) {
    this.location = {
      latitudeRad: D2R * value.latitudeDeg,
      longitudeRad: D2R * value.longitudeDeg
    };
  }
},
```
